tests/features/menus_and_blocks/menus.feature

```diff
@@ -14,16 +14,33 @@ Feature: Menus
       | menu_name          | title           | uri                                               | expanded | parent_uri                          | parent_title    | icon_image    |
       | main               | BEHAT CHILDCARE | base:behat-programs/behat-childcare               | 1        | base:behat-programs                 | BEHAT PROGRAMS  | childcare.png |
       | main               | BEHAT DAYCARE   | base:behat-programs/behat-childcare/behat-daycare | 1        | base:behat-programs/behat-childcare | BEHAT CHILDCARE |               |
-    Then I am an anonymous user
 
   Scenario: Check menu items are in the appropriate place
-    Given I am on the homepage
+    Given I am an anonymous user
+    And I am on the homepage
     Then I should see "BEHAT GIVE" in the "header"
     And I should see "BEHAT PROGRAMS" in the "header"
     And I should see an "ul.nav.dropdown-menu .menu-item-behat-childcare" element
     And Element "ul.nav.dropdown-menu .menu-item-behat-childcare" has text "BEHAT CHILDCARE"
-    And I should see an "ul.nav.dropdown-menu .menu-item-behat-childcare .section-icon" element
     And Element "ul.nav.dropdown-menu .menu-item-behat-childcare" has text "BEHAT DAYCARE"
     And I should see "BEHAT LOCATIONS" in the "footer"
     And I should see "BEHAT JOIN" in the "footer"
     And I should see "BEHAT CAREERS" in the "footer"
+
+  Scenario: Create footer menu left item and check
+    Given I am logged in as a user with the Editor role
+    And I go to "/admin/structure/menu/manage/main/add"
+    And I fill in "Menu link title" with "BEHAT EDITOR ADD"
+    And I fill in "Link" with "http://openymca.org/"
+    And I check the box "Show as expanded"
+    And I attach the file "childcare.png" to "Icon image"
+    And I press "Save"
+    When I am an anonymous user
```

Comment:
Here we can go with selenium. And verify if menu items appear in dropdown with image displayed.